src/vsr.zig

```diff
@@ -638,6 +638,35 @@ pub const UpgradeRequest = extern struct {
     }
 };
 
+/// To ease investigation of accidents, assign a separate exist status for each fatal condition.
```

Comment:
:heart:

Reply:
Typo in `exist status` but I won't bother you with a PR for such a minor change!

Reply:
Thanks, fixed!